components/nexusoperations/config.go

```diff
@@ -34,7 +34,7 @@ var Enabled = dynamicconfig.NewNamespaceBoolSetting(
 	`Enabled toggles accepting of API requests and workflow commands that create or modify Nexus operations.`,
 )
 
-var RequestTimeout = dynamicconfig.NewNamespaceDurationSetting(
+var RequestTimeout = dynamicconfig.NewDestinationDurationSetting(
```

Comment:
I think I missed this in the dynamic config review but can we make the dynamic configs use namespace name and not ID? It's much harder to operate configs by ID.